open_humans/templates/account/signup-form.html

```diff
@@ -98,10 +101,14 @@
 
     <div class="col-sm-8">
       <input type="password" class="form-control" name="password1"
-        id="signup-password" placeholder="Password"
+        id="password" placeholder="Password" data-toggle="password"
```

Comment:
It's unclear to me why this id was renamed?

Reply:
Because I could easily use the library that provides the reveal password functionality.

Reply:
You mean L109? Which could use `#signup-password` instead of `#password`, right?

Reply:
You're right, I don't recall why I did that.  I put it back.